ScrumBoardSimulator/src/main/java/se/bettercode/scrum/gui/ToolBar.java

```diff
@@ -32,22 +32,10 @@ public ToolBar(String[] teams, String[] backlogs) {
 
         teamChoiceBox.setItems(FXCollections.observableArrayList(teams));
         teamChoiceBox.setTooltip(new Tooltip("Select team"));
-        teamChoiceBox.setPrefWidth(100);
 
         backlogChoiceBox.setItems(FXCollections.observableArrayList(backlogs));
         backlogChoiceBox.setTooltip(new Tooltip("Select backlog"));
```

Comment:
Bring this back:         backlogChoiceBox.setPrefWidth(100);

        burnUpChartButton.setPrefSize(150, 20);

        startButton.setPrefSize(100, 20);

        editButton.setPrefSize(100, 20);